Library/Homebrew/test/utils/analytics_spec.rb

```diff
@@ -0,0 +1,93 @@
+require "utils/analytics"
+
+describe Utils::Analytics do
+  describe "::os_prefix_ci" do
+    context "when anonymous_os_prefix_ci is not set" do
+      it "returns OS_VERSION and prefix when HOMEBREW_PREFIX is not /usr/local" do
+        expect(described_class.os_prefix_ci).to include("#{OS_VERSION}, non-/usr/local")
```

Comment:
I think you'll have to override `HOMEBREW_PREFIX` for this test - I'm seeing inconsistent results for this and the other test where you're testing this.